build-tests/x86/ubuntu/test-image-iso-oem-vmx/appliance.kiwi

```diff
@@ -74,6 +77,7 @@
         <package name="isc-dhcp-client"/>
         <package name="netbase"/>
         <package name="dbus"/>
+        <package name="btrfs-progs"/>
```

Comment:
This is required for `dracut-kiwi-lib`. In our spec it is a recommended dependency for Ubuntu, but not a hard requirement. I can't recall why we did this way, I remember we had several issues in the past for Ubuntu, thus probably adding the hard requirement was an issue...not sure, I did not dig into that.